src/components/ChangeCourtSize/ChangeCourtSize.tsx

```diff
@@ -0,0 +1,52 @@
+import { useDisclosure } from "@chakra-ui/react";
+import { Button, Box } from "@chakra-ui/react";
+import { Modal, ModalOverlay, ModalContent } from "@chakra-ui/react";
+import CourtSize from "./CourtSize";
+
+function ScaleFadeEx() {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  return (
+    <>
+      <Button
+        onClick={isOpen ? onClose : onOpen}
+        size="sm"
+        borderRadius="lg"
+        zIndex="popover"
+        bgColor={isOpen ? "CourtSizecolor.btc" : "White"}
+        _after={{
+          content: `${
+            isOpen ? "url(@/assets/svg/en-sort-up.svg)" : "url(@/assets/svg/en-sort-down.svg)"
+          }`,
+          m: "3",
+          mb: "2",
+        }}
+      >
+        Select court size
+      </Button>
```

Comment:
The problem is that after we use a new approach of using SVG files(you can check how to implement it in the sidebar component as an example) The SVG files should be imported and used as a React component.

![image](https://user-images.githubusercontent.com/12379895/168732958-88200c82-1f4c-4ac6-9a03-2c6f9cf9113f.png)

Reply:
Thanks! Tao Zong, I have modified it. I'll do the component testing Once I finished my code